### PR TITLE
select aml supervisor changes

### PR DESCRIPTION
**Pull request by sardhani-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by AmritaSasikumar-CH — COMMENTED** [review]
Please can you add tests for error messages as well

**Review by AmritaSasikumar-CH — COMMENTED** [review]
shouldn't it be CILEX instead of CILEx?

File: src/main/views/common/select-aml-supervisory-bodies/select-aml-supervisory-bodies.njk
Comment (AmritaSasikumar-CH) [inline src/main/views/common/select-aml-supervisory-bodies/select-aml-supervisory-bodies.njk:1]: Please can you cross check the AML bodies listed here with the ticket?

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Pull request merged**